internal/provider/provider.go

```diff
@@ -126,9 +129,25 @@ func createDefaultClient(ctx context.Context, cfg *pconfig.ProviderConfig) (*fab
 	// A value less than zero means there is no cap.
 	fabricClientOpt.Retry.MaxRetryDelay = -1
 
+	ctx, lvl, err := pclient.NewFabricSDKLoggerSubsystem(ctx)
+	if err != nil {
+		tflog.Error(ctx, "Failed to initialize Microsoft Fabric SDK logger subsystem", map[string]any{"error": err.Error()})
+
+		return nil, err
+	}
+
+	if cls := os.Getenv("AZURE_SDK_GO_LOGGING"); cls == "all" && lvl != hclog.Off {
```

Comment:
Would it be possible to move `AZURE_SDK_GO_LOGGING` and `all` to reusable constants, and also do case-insensitive comparisons with the values (people might use `ALL` instead of `all` and expect it to work... although now that I think of it, it may always be set from the other part of the code. Anyway, it would be nice to have a constant at least)

Reply:
done :)